pages/index.tsx

```diff
@@ -112,23 +146,99 @@ const Home: NextPage = () => {
               }
             />
           </div>
-          <button className="w-max p-2.5 px-4 text-gray-600">
-            <FontAwesomeIcon icon={faRedo} />
-          </button>
-        </div>
-        <div className="container mx-auto grid sm:grid-cols-2 lg:grid-cols-3 gap-6">
-          {data?.slice(0, 20).map((font) => (
-            <FontCard
-              key={font.family}
-              data={font}
-              fontSize={fontSize}
-              onClick={(font) => {
-                push(`/speciment/${font.family}`)
+          <div className="p-2 flex flex-col justify-center">
+            <Button
+              icon="redo"
+              onClick={() => {
+                setKeyWord('')
+                setFontSize(40)
+                setPreviewText(
+                  'Almost before we knew it, we had left the ground.'
+                )
+                const previewInputVal = document.getElementById(
+                  'previewInput'
+                ) as HTMLInputElement
+                if (previewInputVal !== null) previewInputVal.value = ''
+                const searchInputVal = document.getElementById(
+                  'searchInput'
+                ) as HTMLInputElement
+                if (searchInputVal !== null) searchInputVal.value = ''
               }}
+              className="h-8 w-8 grid items-center justify-center text-xl rounded-full bg-white text-gray-600 disabled:text-gray-400 hover:bg-gray-50 hover:text-gray-800"
+              disabled={
+                previewText !==
+                  'Almost before we knew it, we had left the ground.' ||
+                fontSize !== 40 ||
+                keyWord !== ''
+                  ? false
+                  : true
+              }
             />
+          </div>
+        </div>
+
+        <div className="p-2 px-14 flex">
+          <h3 className="mr-4 text-blue-600">Categories</h3>
+          {CATEGORIES.map((name, index) => (
+            <div key={index} className="flex items-center mr-4">
+              <input
+                className="mr-1"
+                type="checkbox"
+                id={`checkbox-${index}`}
+                name={name}
+                value={name}
+                checked={cateList[index]}
+                onChange={() => handleOnChangeCheckBox(index)}
+              />
+              <label htmlFor={`checkbox-${index}`}>{formatText(name)}</label>
+            </div>
+          ))}
+        </div>
+
+        <div className="p-2 pb-10 px-14 flex">
+          <h3 className="mr-4 text-blue-600">Languages</h3>
+          <select
+            defaultValue={language}
+            onChange={({ target: { value: val } }) => setLanguage(val)}
+          >
+            {LANGUAGES.map((name, index) => (
+              <option
+                key={index}
```

Comment:
If name is unique, consider using it instead

```suggestion
            {LANGUAGES.map((name, index) => (
              <option
                key={name}
```